src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -186,11 +186,12 @@ public interface DsVersionHandler<T> {
     @GET
     @AuthRequired
     @Path("{id}")
-    public Response getDataset(@Context ContainerRequestContext crc, @PathParam("id") String id, @Context UriInfo uriInfo, @Context HttpHeaders headers, @Context HttpServletResponse response) {
+    public Response getDataset(@Context ContainerRequestContext crc, @PathParam("id") String id, @Context UriInfo uriInfo, @Context HttpHeaders headers, @Context HttpServletResponse response,  @QueryParam("returnOwners") Boolean returnOwners) {
```

Comment:
For the frontend, we will need the same query parameter but in a different endpoint, getVersion, as this is the one we are using to retrieve a dataset. Thx!

```
@Path("{id}/versions/{versionId}")
 public Response getVersion
```

Reply:
@MellyGray do we also need it for getPrivateUrlDatasetVersion @Path("privateUrlDatasetVersion/{privateUrlToken}")?

Reply:
@sekmiller Yes, I think so. Thank you for noting this change!